deploy/charts/emqx-enterprise/templates/StatefulSet.yaml

```
@@ -107,13 +107,13 @@ spec:
             containerPort: {{ .Values.emqxConfig.EMQX_LISTENERS__WSS__DEFAULT__BIND | default 8084 }}
           - name: dashboard
             containerPort: {{ .Values.emqxConfig.EMQX_DASHBOARD__LISTENER__HTTP__BIND | default 18083 }}
-          {{- if not (empty .Values.emqxConfig.EMQX_LISTENERS__TCP__DEFAULT) }}
+          {{- if not (empty .Values.emqxConfig.EMQX_LISTENERS__TCP__LOCAL__BIND) }}
```

maybe keep it more consistent with 4.x and the yaml name `internalmqtt`.
```suggestion
          {{- if not (empty .Values.emqxConfig.EMQX_LISTENERS__TCP__INTERNAL__BIND) }}
```

---

Sure, I can also change this on emqx chart.